ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/AmountMapper.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDAmount;
+import uk.gov.hmcts.cmc.ccd.domain.CCDAmountRange;
+import uk.gov.hmcts.cmc.ccd.exception.MappingException;
+import uk.gov.hmcts.cmc.domain.models.amount.Amount;
+import uk.gov.hmcts.cmc.domain.models.amount.AmountRange;
+import uk.gov.hmcts.cmc.domain.models.amount.NotKnown;
+
+import static uk.gov.hmcts.cmc.ccd.domain.AmountType.NOT_KNOWN;
+import static uk.gov.hmcts.cmc.ccd.domain.AmountType.RANGE;
+
+@Component
+public class AmountMapper implements Mapper<CCDAmount, Amount> {
+
+    private AmountRangeMapper amountRangeMapper;
+
+    public AmountMapper(final AmountRangeMapper amountRangeMapper) {
+        this.amountRangeMapper = amountRangeMapper;
+    }
+
+    @Override
+    public CCDAmount to(Amount amount) {
+        CCDAmount.CCDAmountBuilder builder = CCDAmount.builder();
+        if (amount instanceof AmountRange) {
+            builder.type(RANGE);
+            AmountRange amountRange = (AmountRange) amount;
+            builder.amountRange(amountRangeMapper.to(amountRange));
+        } else if (amount instanceof NotKnown) {
+            builder.type(NOT_KNOWN);
+        }
+
+        return builder.build();
+    }
+
+    @Override
+    public Amount from(CCDAmount ccdAmount) {
+
+        switch (ccdAmount.getType()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1301)